docs/language/internals.rst

```diff
@@ -436,10 +436,3 @@ Our final version of ``nif``, built with ``defmacro/g!`` becomes::
 
 Checking Macro Arguments and Raising Exceptions
 -----------------------------------------------
-
-
-
-Hy Compiler Built-Ins
-=====================
-
-.. TODO: Write this.
```

Comment:
This leaves an empty subheading in the docs. It should also probably get cleaned up.

Reply:
Good point. Done.